app/api/v1/websocket/feedme_websocket.py

```
@@ -0,0 +1,493 @@
+
+@router.get("/feedme/websocket/metrics", response_model=WebSocketMetrics)
+async def get_websocket_metrics(
+    current_user: str = Depends(get_current_user_from_token)
+):
+    """Get current WebSocket system metrics"""
+    try:
+        metrics = realtime_manager.get_metrics()
+        return metrics
+    except Exception as e:
+        logger.error(f"Failed to get WebSocket metrics: {e}")
+        raise HTTPException(status_code=500, detail="Failed to retrieve metrics")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)